README.md

```diff
@@ -1 +1,143 @@
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 portfolio <a name="about-project"></a>
+This pull is created to add javascript to my first capstone project and make interaction on my menu bars and times 
+This PR updates the responsive desktop design for our website to improve its usability on larger screens. The goal of this update is to provide a better user experience for desktop users and ensure that our website looks great on all devices
+
+## 🛠 Built With <a name="built-with"></a>
+<ul>
+  <li>HTML5</li> 
+  <li>CSS3</li>
+  <li>jJavaScript</li>
+  <li>flexbox</li>
+  <li>CSS Grid</li>
+</ul>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="#">n/a</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Updated the HTML and CSS files to improve the responsiveness of the website on larger screens.
+- Implemented a new layout technique to better optimize the website for desktop screens.
+- Improved the typography and spacing of the website to enhance its visual appeal.
+- I also used CSS media queries to adjust the layout of the website based on the screen size.
+- I Used basic JavaScript syntax.
+- I Used JavaScript to manipulate DOM elements.
+- I Used JavaScript events.
+ -I used objects to store and access data.
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- vs code 
+- github
+- laptop
+
+### Setup
+
+Clone this repository to your desired folder:
+
+  - cd my-folder
+  - git clone https://github.com/massazoum/capstone-project-1.git
+  
+  - git clone
+  - npm install or npm init -y
+
+### Run tests
+
+This website has been completed and deployed
+
+### Deployment
+
+  The changes will be deployed to the website through our standard deployment process.
+- [PortFolio](https://massazoum.github.io/Portfolio-setup-and-mobile-first/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Massa zoumanigui**
+
+- GitHub: [@massazoum](https://github.com/massazoum)
+- Twitter: [@zoumaniguimass1](https://twitter.com/zoumaniguimass1)
+- LinkedIn: [LinkedIn](www.linkedin.com/in/massa-zoumanigui-1aba4525a)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Animation**
+- [ ] **Responsive**
+- [ ] **include Javascript**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] As per the project requirements you need to give credit for the owner of the design you are following and it is more professional.

- Example
![image](https://user-images.githubusercontent.com/60083437/227040961-fb8519ce-3b4f-4dde-85b0-f338232b9616.png)